models/migrations/v1000.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
File name? Should be v210.go

Reply:
@Gusted the migration file was renamed as suggested. And tests added to make the implementation final.

---
[source](https://lab.fedeproxy.eu/forgefriends/forgefriends/-/merge_requests/36#note_935)